ITEST/server-core/src/org/openl/itest/core/HttpClient.java

```diff
@@ -50,6 +51,7 @@
 
     private final RestTemplate rest;
     private final URL baseURL;
+    private String cookie;

```

Comment:
Make it as ```ThreadLocal```. We have multithread tests in https://github.com/openl-tablets/openl-tablets/tree/master/ITEST/itest.ws-rest-rules-deploy .